spec/controllers/index_spec.rb

```diff
@@ -0,0 +1,43 @@
+require_relative '../spec_helper'
+
+describe 'Index Controller' do
+
+  it 'should process get / route' do
+    get '/'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include("Welcome to the Sea Lions Den.")
+  end
+
+  it 'should process get /sealions route' do
+    get '/sealions'
+    expect(last_response).to be_ok
+    expect(last_response.body).to include("Let's welcome the new cubs")
+  end
+
+  it 'should process post /sealions route' do
+    post '/sealions'
+    expect(last_response).to be_redirect
+    follow_redirect!
+    expect(last_response.body).to include("Let's welcome the new cubs:")
+  end
+
+  it 'should process get /sealions/:id route' do
+    get '/sealions/:id'
```

Comment:
Look at your get route here! You are not actually sending in a parameter. You have to format it like get '/sealions/<some known sealion id>'